capi/geos_c.h.in

```diff
@@ -3461,6 +3475,66 @@ extern GEOSGeometry GEOS_DLL *GEOSConcaveHull(
     double ratio,
     unsigned int allowHoles);
 
+/**
+* Computes a boundary-respecting hull of a polygonal geometry,
+* with hull shape determined by a target parameter
+* specifying the fraction of the input vertices retained in the result.
+* Larger values compute less concave results.
+* A value of 1 produces the convex hull; a value of 0 produces the original geometry.
+* An outer hull is computed if the parameter is positive,
+* an inner hull is computed if it is negative.
+*
+* \param g the polygonal geometry to process
+* \param vertexNumFraction the target fraction of number of input vertices in result
+* \return A newly allocated geometry of the concave hull. NULL on exception.
+*
+* Caller is responsible for freeing with GEOSGeom_destroy().
+* \see geos::algorithm::hull::PolygonHull
+*/
+extern GEOSGeometry GEOS_DLL *GEOSPolygonHull(
+    const GEOSGeometry* g,
+    double vertexNumFraction);
+
+
+/**
+* Constructs a concave hull of a set of polygons, respecting
+* the polygons as constraints.
+*
+* A concave hull is a possibly non-convex polygon containing all the input polygons.
+* A given set of polygons has a sequence of hulls of increasing concaveness,
+* determined by a numeric target parameter.
+* The computed hull "fills the gap" between the polygons,
+* and does not intersect their interior.
+*
+* The concave hull is constructed by removing the longest outer edges
+* of the Delaunay Triangulation of the space between the polygons,
+* until the target criterion parameter is reached.
+*
+* "Maximum Edge Length" is the length of the longest edge between the polygons is no larger than this value.
+*
+* \param g the valid multi-polygonal geometry to process
```

Comment:
```suggestion
* \param g the valid MultiPolygon geometry to process
```

Reply:
except that this accepts Polygons as well.  Should probably be "polygonal".  (Grrr that OGC SFS has such a awkward geometry class hierarchy)